common-design/sass/cd-layout/_cd-layout.scss

```diff
@@ -39,6 +38,14 @@ body {
 
 }
 
+@media (min-width: map-get($cd-breakpoints, desktop)) {
```

Comment:
not for this PR but let's totally refactor this to reduce the syntax and avoid writing so much for a little ol' MQ:

```scss
@include mq-desktop() {
  // desktop styles
}
```

Reply:
Split out into CD-37